depscan/lib/csaf.py

```diff
@@ -1313,6 +1313,8 @@ def parse_cvss(ratings):
     """
     if not ratings or not (vector_string := ratings[0].get("vector")):
         return {}
+    if not vector_string or vector_string == "None":
+        return {}
     try:
         cvss_v3 = cvss.CVSS3(vector_string)
         cvss_v3.check_mandatory()
```

Comment:
`except [CVSSError, ValueError]:` - Getting some error related to BaseException due to this line.

Reply:
@prabhu It should be `except (CVSSError, ValueError):`